### PR TITLE
2 srijan

### PR DESCRIPTION
**Pull request by SrijanBajracharya**
#5 Multiples of 3 and 5

### CODE REVIEW

File: src/com/lftechnology/java/training/srijan/FirstTutorial.java [hunk 1]
```diff
@@ -0,0 +1,30 @@
+package com.lftechnology.java.training.srijan;
+
+import java.util.Scanner;
+
+public class FirstTutorial {
```
Comment (nimesh-mishra) [inline src/com/lftechnology/java/training/srijan/FirstTutorial.java:5]: Use meaningful class name.

File: src/com/lftechnology/java/training/srijan/basics/Multiples.java [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+	/**
+	 * <p>
+	 * Prints numbers between 1 and 1000 divisible by 3 and 5
+	 * </p>
+	 * <p>
+	 * Calculates sum of numbers exactly divisible by 3 and 5 and shows result
+	 * </p>
+	 * 
+	 * @param num1
```
Comment (nimesh-mishra) [inline src/com/lftechnology/java/training/srijan/basics/Multiples.java:28]: State the data type of the parameters

File: src/com/lftechnology/java/training/srijan/basics/Multiples.java [hunk 3]
```diff
@@ -0,0 +1,50 @@
+package com.lftechnology.java.training.srijan.basics;
+
+/**
+ * If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9. The sum of these multiples is 23.
+   Find the sum of all the multiples of 3 or 5 below 1000.
```
Comment (utsabneupane) [inline src/com/lftechnology/java/training/srijan/basics/Multiples.java:5]: include `@author` in class level documentation also

File: src/com/lftechnology/java/training/srijan/basics/Multiples.java [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+ */
+
+import java.util.logging.Logger;
+
+public class Multiples {
+	private int num = 1000;
+	private int sum = 0;
+	
+	private static final Logger LOGGER = Logger.getLogger(Fabonacci.class.getName());
```
Comment (Msanjib) [inline src/com/lftechnology/java/training/srijan/basics/Multiples.java:14]: What is Fabonacci.class.getName().Use the same class name you are using the logger for.

File: src/com/lftechnology/java/training/srijan/basics/Multiples.java [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+	public static void calculateDivisibles(int num1, int num2) {
+		Multiples multiple = new Multiples();
+		for (int i = 1; i < multiple.num; i++) {
+			if (i % num1 == 0 || i % num2 == 0) {
+				LOGGER.info(i + " ");
+				multiple.sum = multiple.sum + i;
+			}
+		}
+		LOGGER.info("\n");
+		LOGGER.info("Sum of Multiples are=" + multiple);
```
Comment (utsabneupane) [inline src/com/lftechnology/java/training/srijan/basics/Multiples.java:43]: can include both logger into a single one like `LOGGER.info("\nSum of Multiples are=" + multiple);` if required

File: src/com/lftechnology/java/training/srijan/basics/Multiples.java (hunk 4 above)
Comment (bhuwang) [inline src/com/lftechnology/java/training/srijan/basics/Multiples.java:14]: Copy & Paste effect :-1:

File: src/com/lftechnology/java/training/srijan/FirstTutorial.java [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+public static void main(String[] args){
+		
+		Scanner input = null;
+		
+		try{
+			input = new Scanner(System.in);
+			System.out.println("Enter your Name with Salutation");
+			String nameWithSalutation = input.nextLine();
+			nameWithSalutation=nameWithSalutation.trim();
```
Comment (Msanjib) [inline src/com/lftechnology/java/training/srijan/FirstTutorial.java:14]: these two statements could be combined input.nextLine().trim();

File: src/com/lftechnology/java/training/srijan/basics/Multiples.java (hunk 3 above)
Comment (Msanjib) [inline src/com/lftechnology/java/training/srijan/basics/Multiples.java:5]: This document should be just above the class name

File: src/com/lftechnology/java/training/srijan/FirstTutorial.java [hunk 7]
```diff
@@ -0,0 +1,30 @@
...
+			
+			String[] text = nameWithSalutation.split(" ");
+			System.out.println("Salutation:" + text[0]);
+			//System.out.println("Name:" + text[1] +" "+ text[2]);
+			System.out.println("Name:");
+			for(int i=1;i<text.length;i++){
+				System.out.print(text[i] +" ");
+			}
+		}catch(Exception e){
+			System.out.println("There is an exception" + e);
```
Comment (utsabneupane) [inline src/com/lftechnology/java/training/srijan/FirstTutorial.java:25]: print `e.getMessage()` instead of printing while `e`

**Pull request merged**